### PR TITLE
Raid UI patch

### PR DESCRIPTION
**Pull request by kcper**
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?
Mark signed up items
image

Fix weapon not showing up in rewards modal
image

Prevent joining when they can't, show error
image

Clarify the XP will be claimed automatically
image

Replace "bounty chest" icon (that is used for key lootbox nft and was used as junk placeholder in drops) with junk armor
image

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.
### PR Description

Changes regarding to feedback from early-testers-chat.
This PR also attempts to fix problems with choosing weapons, which only happen on testnet (never happened and could not reproduce it locally).

I'm not sure if raid1.sol changes need another migration which I think would be the exact copy of the previous one.

### CODE REVIEW

File: frontend/src/components/CharacterArt.vue [hunk 1]
```diff
@@ -231,6 +231,7 @@ export default {
       this.scene.add(light);
     },
     setupModel() {
+      if(this.scene === null || this.scene === undefined) return;
```
Comment (seiyria) [inline frontend/src/components/CharacterArt.vue:234]: is there a reason this can't be `if(!this.scene)`?

File: frontend/src/views/Raid.vue [hunk 2]
```diff
@@ -256,26 +273,37 @@ export default {
...
 
+      const canJoin = await this.canJoinRaid(this.currentCharacterId, this.selectedWeaponId);
+      if(!canJoin) {
+        this.$dialog.notify.error(`Can't join... Make sure a raid is active, your stamina/durability are both > 0
+          and selected character/weapon are not locked in the raid already.`);
+        return;
+      }
+
       try {
         console.log('Trying to join raid...');
```
Comment (seiyria) [inline frontend/src/views/Raid.vue:284]: conosle.log

File: frontend/src/views/Raid.vue [hunk 3]
```diff
@@ -256,26 +273,37 @@ export default {
...
         await this.joinRaid({ characterId: this.currentCharacterId, weaponId: this.selectedWeaponId});
+        this.selectedWeaponId = null;
         console.log('Made it to the other side at least...');
```
Comment (seiyria) [inline frontend/src/views/Raid.vue:287]: console.log

Comment (kcper) [reply frontend/src/views/Raid.vue:287]: Wasn't added by me, but I'll delete all other raid realted console logs then.

File: frontend/src/components/CharacterArt.vue (hunk 1 above)
Comment (kcper) [reply frontend/src/components/CharacterArt.vue:234]: Nope, not sure why I did this way.

File: frontend/src/views/Raid.vue (hunk 3 above)
Comment (seiyria) [reply frontend/src/views/Raid.vue:287]: I'm not sure if they're needed for debugging or what, but unless they're needed I'd like to keep them away. thanks!

**Review by poshdan — APPROVED** [review]
looks good now. Gonna add my trinket star rarity tweak, run a test locally then deploy to testnet probably. Maybe we can get it on mainnet tomorrow if no more glaring UX issues exist

**Pull request merged**